### ISSUE TITLE
breadcrumbs: missing a11y text for collapsed button

### ISSUE THREAD

**Issue by saiponnada** [eBay/ebayui-core#1987]
# Bug Report

## eBayUI 

https://opensource.ebay.com/ebayui-core/?path=/story/navigation-disclosure-ebay-breadcrumbs--heading

## Description

The fake menu button doesn't have a11y text associated with it during collapsed state.

## Screenshots

![image](https://github.com/eBay/ebayui-core/assets/6342519/1419d125-c4a3-408c-9862-2768851a4955)
